tests/unit/fancyrag/kg/test_phases.py

```
@@ -521,6 +542,8 @@ def test_helpers_do_not_touch_environment(monkeypatch, tmp_path):
         shared_client_factory=lambda settings: object(),
         embedder_factory=lambda client, settings: object(),
         llm_factory=lambda client, settings: object(),
+        semantic_llm_factory=lambda client, settings, *, schema: object(),
```

Ruff (0.14.10)

Unused lambda argument: `client`

(ARG005)